pkg/controller/servicemesh/controlplane/profiles_test.go

```diff
@@ -160,6 +160,37 @@ func TestProfiles(t *testing.T) {
 				smcp.Spec.TechPreview.RemoveField("wasmExtensions")
 			},
 		},
+		{
+			name: "MAISTRA-2409",
+			v:    versions.V2_0,
```

Comment:
do we need to duplicate this test with a 2.1 version?

Reply:
Not really, as the code is the same for both 2.0 and 2.1. If anything, we'd have to extend all the tests to check for both 2.0 and 2.1, but that's a whole new PR.

Reply:
The code is the same, but the charts not.